integration_testing/features/admin/admin-change-language.feature

```diff
@@ -4,6 +4,20 @@ Feature: Admin changes user interface language
   Background:
     Given I am signed in to Kolibri as facility admin user
 
+  Scenario: Admin has changed language from <device_language> to <language> prior to logging in
+    When I log in
+    Then Kolibri is in <language>
+    When I log out
+    Then I am redirected to the sign in page and Kolibri is in <language>
+    When I open a new tab and open Kolibri
+    Then Kolibri is shown in <language>
+    When I refresh the page
+    Then Kolibri is shown in <language>
+    When I navigate between and within Coach, Learn, and Facility sections of Kolibri
+    Then Kolibri remains in <language>
+    When I close my browser altogether and go to Kolibri's root server URL
```

Comment:
@nucleogenesis In my VM at least, both FF & Chrome browsers retain the languages setting used by the user who last visited Kolibri, even when it's different from the <device_language>. So atm, the only way to actually get Kolibri to display in the set <device_language>, is to open the incognito window. For more straight testing, I suggest you change the above `When` accordingly, as just closing, reopening the browser and going to the root URL won't do the trick :wink:.

Reply:
Do the above in all 4 scenarios.